src/MoBi.Core/Services/SimulationLoader.cs

```diff
@@ -48,14 +49,14 @@ private static MoBiMacroCommand createLoadCommand(IMoBiSimulation simulation)
          };
       }
 
-      private void addSimulationToProject(IMoBiSimulation simulation, MoBiMacroCommand loadCommand)
+      private void addSimulationToProject(IMoBiSimulation simulation, MoBiMacroCommand loadCommand, bool shouldCloneSimulation)
       {
          var moBiSimulation = simulation;
 
          simulation.MoBiBuildConfiguration.AllBuildingBlockInfos().Each(checkTemplateBuildingBlock);
          var project = _context.CurrentProject;
 
-         if (!project.IsEmpty)
```

Comment:
I also changed this call to ensure that a clone was only done if needed. Project with building blocks are ok for example